src/BookingConflictConditionFactory.php

```diff
@@ -126,27 +169,27 @@ public function make($config = null)
             );
         }
 
+        // For each non-blocking status, AND the condition to a "NOT EQUAL TO" sub-condition for that status
+        // This makes the condition disregard bookings with these statuses
+        foreach ($this->_getNonBlockingStatuses() as $nonBlockingStatus) {
+            $b->and(
```

Comment:
Here, you are creating an `AND` condition. But it does not appear like they would apply, because they don't seem to be recorded anywhere.

Reply:
My bad.